playbooks/roles/ocp-pod-disruption-budget/templates/pdb-deployment.yml.j2

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ pdb_name }}
+  namespace: "{{ pdb_namespace }}"
+spec:
+{% if pdb_minAvailable and pdb_maxUnavailable %}
```

Comment:
playbook should probably fail or warn the user if both values are set together which is not allowed.

Reply:
Added the play to check provided values for minAvailable and maxUnavailabe

Reply:
this check should be removed then.

Reply:
This check has been removed. Also, added check for both present and blank values.